src/main.rs

```diff
@@ -34,6 +44,219 @@ pub mod test_harness;
 pub static SECP: Lazy<::bitcoin::secp256k1::Secp256k1<::bitcoin::secp256k1::All>> =
     Lazy::new(::bitcoin::secp256k1::Secp256k1::new);
 
-fn main() {
-    println!("Hello, world!");
+mod maker {
+    use super::*;
+    use comit::LocalSwapId;
+
+    #[derive(Debug, PartialEq)]
+    pub enum New_Order {
+        Created(Order),
+    }
```

Comment:
Why isn't this just a newtype? :)

```rust
pub struct OrderCreated(Order);
```

If you need an enum with more events at some points, you can always do this:

```rust
enum Event {
	OrderCreated(OrderCreated)
}
```